### DISCUSSION TITLE
How to upload generated build outputs to Vercel?

### DISCUSSION THREAD

**Discussion by ritiksharmarj** [vercel/community#1870] [Help]
### Summary

I have created a react app and deployed it to vercel but I don't want to show my source code file to anyone that's why I have generated a build but how I can upload this generated build outputs to Vercel?

I have tried using "vercel deploy --prebuilt" but got error.

### Additional information

Error: The "--prebuilt" option was used with the target environment "preview", but the prebuilt output found in ".vercel/output" was built with target environment "production". Please run `vercel --prebuilt --prod`.

**Comment by TooTallNate** [comment]
The error message tells you what to run instead 😃 

> Please run `vercel --prebuilt --prod`.

**Reply by ritiksharmarj** [reply]
I already tried but don't work as supposed to be. I am still able to see my source code on my live site which I don't want that's why I want to upload the build folder.

In netlify there is an upload file option but here it isn't.
